### PR TITLE
data: NikhilRaikwar

### PR DESCRIPTION
**Pull request by NikhilRaikwar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4812"></a>

### CODE REVIEW

File: data/NikhilRaikwar.json [hunk 1]
```diff
@@ -1,36 +1,85 @@
...
+      "description": "Started creating content about technology & provide knowledge about programming",
+      "url": "https://www.youtube.com/@nikhilaicodes6700"
+    },
+    {
+      "title": "Started GitHub",
+      "date": "2020",
+      "icon": "FaGithub",
+      "color": "green",
+      "description": "Started Github first time using to host projects & Rest is History.",
+      "url": "https://github.com/eddiejaoude"
```
Comment (Panquesito7) [inline data/NikhilRaikwar.json:82]: You might want to update this link.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by NikhilRaikwar — COMMENTED** [review]
I change the link & pls approve the changes

File: data/NikhilRaikwar.json (hunk 1 above)
Comment (NikhilRaikwar) [reply data/NikhilRaikwar.json]: i update it

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by yashdev9274 — APPROVED** [review]
lgtm 

@NikhilRaikwar next time when you create PR do mention the changes you made while creating PRs not

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**